lib/alephant/renderer/error/invalid_base_path.rb

```diff
@@ -1,9 +1,19 @@
+require 'alephant/logger'
+
 module Alephant
   module Renderer
     module Error
       class InvalidBasePath < StandardError
+        include Logger
+
         def initialize(path)
           super "Path '#{path}' to renderer/queue directory is invalid"
+
+          logger.metric('ViewMapperInvalidPath')
+          logger.error(
+            event:  :ViewMapperBasePathInvalidFound,
```

Comment:
I think this is naming, InvalidFound sounds a bit odd. ViewMapperInvalidBasePath?

Also double spacing between keys and values.

Reply:
i'll set the event to `:ViewMapperInvalidPath` to match the metric